src/content/popup/lang-tag.ts

```diff
@@ -1,5 +1,8 @@
 import browser from 'webextension-polyfill';
 
+// TODO: this needs to be reworked to work in a non-extension context (for testing in Cosmos)
+// Not entirely sure how to make that happen with the import above
```

Comment:
Basically, any file that has that import can't be used in a non-extension context so you end up passing in `browser.i18n.getMessage.bind(browser.i18n)` as a parameter instead.

In this case, however, I think it might make sense to add `langTag` to `i18nContentType` and then set up a `useMemo` in there that calls `t('lang_tag')` any time `t` or `locale` changes. That way we won't need to import `lang-tag.ts` at all.

Reply:
Thanks for the help with that. I had a stab at doing what you said, but let me know if I did it right 😅. I am woefully under-experienced on the i18n front.

I think with these changes we can delete the `lang-tag.ts` file entirely, and just replace all `getLangTag()` calls with the `locale` of `i18nContextType`, no?

Reply:
Sorry, I got this wrong. I thought the locale names we used match the BCP-47 lang tags but in the case of simplified Chinese they're different (`zh_hans` for the locale name vs `zh-Hans` for the lang tag). As a result I think we need to look up `t('lang_tag')` (presumably inside a `useMemo` hook) and return that instead of `locale`.

Reply:
I think we can drop this comment now.